deploy/ansible/inventory/hosts_standalone_nodes.list

```diff
@@ -0,0 +1,16 @@
+# This is sample inventory file for deployment using docker-compose (multiple nodes of iroha on target hosts)
+# To start with:
+# 1) set group name (not mandatory, you can use already predefined): e.g. [iroha-nodes].
+# 2) change inventory hostname (not mandatory, you can use already predefined): e.g. iroha-1.
+# 3) set IP address to the address of your server ("ansible_host" variable)
+# 4) set "ansible_user", or leave "root"
+# 5) set "key" variable in the following way:
+# key=0 for the first host of the first node in the inventory
+# for next host set key=key[prev_host] + 1
+# NOTE: 1) you can create more than one group
+# NOTE: 2) you can set unique value of "nodes_in_group" for each host by setting it in
```

Comment:
"...by setting it in" where?

Reply:
@bakhtin thanks for that. It was a big fail from my side.